src/plugins/intel_cpu/src/nodes/mkldnn_rnn.cpp

```diff
@@ -725,21 +717,67 @@ void MKLDNNRNN::createDescriptor(const std::vector<MemoryDescPtr> &inputDesc,
     supportedPrimitiveDescriptors.emplace_back(config, ref_any);
 }
 
-void MKLDNNRNN::createPrimitive() {
+void MKLDNNRNN::prepareParams() {
+    auto dataMemPtr = getParentEdgesAtPort(0).front()->getMemoryPtr();
```

Comment:
please also validate other non const inputs and check `GetPrimitivePtr`
https://github.com/openvinotoolkit/openvino/blob/132b755a469af94213485fea508b4a8e7db1ef60/inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp#L930-L944

Reply:
Done.